Doc updates for updated PowerShell module (#267)
microsoft/winget-cli-restsource: Tools/PowershellModule/doc/PowerShell/Add-WinGetManifest.md

### -Path
Supports input from pipeline.
The path to the Manifest file or folder hosting either a JSON or YAML file(s) that will be uploaded to the REST source. 

> This path may contain a single Manifest file, or a folder containing Manifest files for a single Manifest. Does not support 

Reviewer: ```suggestion
This path may contain a single Manifest file, or a folder containing multiple files for a single Manifest.

```
